.buildkite/pipeline.yml

```diff
@@ -31,6 +31,8 @@ steps:
       - 'ln -s /var/lib/fc-ci/vmlinux.bin testdata/vmlinux'
       - 'ln -s /usr/local/bin/firecracker testdata/firecracker'
       - 'ln -s /usr/local/bin/jailer testdata/jailer'
-      - "make test EXTRAGOARGS='-v -count=1'"
+      - 'chown 999:999 testdata/vmlinux'
+      - 'chown 999:999 testdata/root-drive.img'
+      - "sudo TMPDIR=/local/tmp make test EXTRAGOARGS='-v -count=1'"
```

Comment:
Please see my previous comment.  I'd like to have this addressed before we merge.